### PR TITLE
FIX: Crash due to actionsheet changes

### PR DESCRIPTION
**Pull request by marcosrdz**
Ill write e2e next

### CODE REVIEW

File: screen/wallets/transactions.js [hunk 1]
```diff
@@ -404,70 +409,38 @@ const WalletTransactions = ({ navigation }) => {
...
+      },
+      buttonIndex => {
+        switch (buttonIndex) {
+          case 0:
+            break;
+          case 1:
+            choosePhoto();
+            break;
+          case 2:
+            scanQrHelper(navigate, onBarCodeRead);
```
Comment (Overtorment) [inline screen/wallets/transactions.js:434]: this is not how scanQrHelper works. function signature is:

```
scanQrHelper(
  navigateFunc: (scr: string | any, params?: any) => void,
  currentScreenName: string,
  showFileImportButton = true,
  onDismiss?: () => void,
): Promise<string | null>
```

**Review by Overtorment — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xdq5omxhkmg5s2ulqpftqcljy4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/778)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/mbeuxby2rjyfc6ydvsklgl6xsi?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/779)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/cdotomi7ygsuodlvxksr4f7dcy?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/784)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**